lib/players/human.rb

```diff
@@ -0,0 +1,16 @@
+module Players
+  class Human < Player #inherits from Player
+
+    attr_reader :input, :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def move(board) #asks the user for input and returns it
+      puts "#{token}: Please select an open position from 1 to 9:"
+      position = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.strip
end
```